### ISSUE TITLE
Where is your default startup screen settings？

### ISSUE THREAD

**Issue by YuanPres** [purcell/emacs.d#452]
I‘m Very sorry, trouble you again.

I am now very annoying to "* scratch *" as the first show after startup. 
I searched your configuration, but because of my limited ability and did not see this setting in there.
So...Where ?

And....

I tried to open "use-dialog-box" in your file: `init-gui-frames.el`. But no effect. This is my code:
`(setq use-dialog-box t)`
But its value is still "nil"
So what's wrong ？Is there anything that covers this setting?

Thanks.

**Comment by purcell** [comment]
Hi. I'm really sorry, but my time is extremely limited and I can't provide support in the form of answering lots of little questions like this: at a certain point, you'll need to investigate more deeply and look elsewhere for general Emacs answers.

**Comment by YuanPres** [comment]
Okay ... I think I need to change a more simple configuration.
I **do not** want to trouble you,
But your configuration in many places are different from the general information I found. And you don't seem to have a structured description of your configuration differences ...
I've seen your description of your function, but I still do not know "what's set up somewhere" ... it should be said that I know some, but not all. You seem to be in accordance with the programming language, rather than in accordance with the function to structure this configuration.
So I was so confused.

In short, thank you for your warm help, and......

I am so sorry to bring you troubled and take up your time!

**Comment by Zalafina** [comment]
@YuanPres 
I have tried to set "initial-scratch-message" to nil in the "init-lisp.el" for startup from a blank frame screen. You may try it.
The Code below is the modification from "git diff" command

`--- a/lisp/init-lisp.el
+++ b/lisp/init-lisp.el
@@ -5,9 +5,10 @@
 
 (require-package 'lively)
 
-(setq-default initial-scratch-message
-              (concat ";; Happy hacking, " user-login-name " - Emacs ♥ you!\n\n"))
+;;;(setq-default initial-scratch-message
+;;;              (concat ";; Happy hacking, " user-login-name " - Emacs ♥ you!\n\n"))
 
+(setq-default initial-scratch-message nil)
 
 ^L
 ;; Make C-x C-e run 'eval-region if the region is active
`

**Comment by YuanPres** [comment]
@Zalafina 
Thanks, 
But I don't want to startup from a blank frame screen. I want to startup from I set it myself.

**Comment by Zalafina** [comment]
Purcell's startup message is set like this:
```
(setq-default initial-scratch-message
             (concat ";; Happy hacking, " user-login-name " - Emacs ♥ you!\n\n"))

```

Just modify it as you want.

**Comment by YuanPres** [comment]
@Zalafina 
Thank you, but I already know that these places can be modified.
I want to change the "default startup pages". It's not "default startup message" .
You don‘t need to help me because I'm reading purcell's code and trying to find these things.

Thank you very much, and sorry for my English.

**Comment by JiaHong-Lee** [comment]
@ReimuXMX 
Perhaps changing the initial-buffer-choice will help?  I put this code into my init.el file and so everytime emacs starts, it opens the indicated file.
```
;; DEFAULT BUFFER CHOICE
(setq initial-buffer-choice
      "c:/Users/Lee Jia Hong/Desktop/projects_and_assignments.txt")
```

Hope it helps. Thanks!

**Comment by ReimuXMX** [comment]
@JiaHong-Lee 
Thanks,
I have thought of this solution, but it did not work,
I put this code into all the code before loading ... ...
it works.
